### PR TITLE
[bitnami/schema-registry] Fixed SS init script to copy ssl.keystore and ssl.trustore to correct folder for HTTPS setup

### PR DESCRIPTION
**Pull request by ETisREAL**
As highlighted in the issue #21791 it was necessary to update the Stateful Set init script to copy the ssl.keystore.jks and ssl.trustore.jsk files (if present) over to `/opt/bitnami/schema-registry/certs/` to enable HTTPS setup

### CODE REVIEW

File: bitnami/schema-registry/Chart.yaml [hunk 1]
```diff
@@ -34,4 +34,4 @@ maintainers:
 name: schema-registry
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/schema-registry
-version: 16.8.0
+version: 17.0.0
```
Comment (Mauraza) [inline bitnami/schema-registry/Chart.yaml:37]:
```suggestion
version: 16.8.1
```

**Review by Mauraza — CHANGES_REQUESTED** [review]
Hi @ETisREAL,

Thanks for the contribution!!!! I think it is not necessary to upgrade to major, it is not a breaking change.

File: bitnami/schema-registry/Chart.yaml (hunk 1 above)
Comment (ETisREAL) [reply bitnami/schema-registry/Chart.yaml]: Sure thing :)

**Review by Mauraza — APPROVED** [review]
LGTM!

**Comment by ETisREAL** [comment]
Thank you @Mauraza :) Have a nice WE

**Pull request merged**